### PR TITLE
remove block view! 

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

**Comment by debris** [comment]
I decided to remove `view!` in several chunk as this pr would get to big to review it properly.

**Comment by dvdplm** [comment]
Can you tell us why you want to remove it? What's wrong with it?

**Comment by debris** [comment]
it unsafely `unwrap` rlp. We have a bunch of issues related to that

File: ethcore/src/block.rs [hunk 1]
```diff
@@ -637,21 +637,22 @@ mod tests {
 	use ethereum_types::Address;
 	use std::sync::Arc;
 	use transaction::SignedTransaction;
+	use verification::queue::kind::blocks::Unverified;
 
 	/// Enact the block given by `block_bytes` using `engine` on the database `db` with given `parent` block header
 	fn enact_bytes(
-		block_bytes: &[u8],
+		block_bytes: Vec<u8>,
```
[the remark this answers is not in the document]
Comment (ascjones) [reply ethcore/src/block.rs]: Looks like test functions

[the remark this answers is not in the document]
Comment (niklasad1) [reply ethcore/src/block.rs]: ah lol thanks, never mind :smiley:

**Pull request merged**